packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -845,7 +845,7 @@ export class InputDatePicker
       focusTrapEl: el,
       focusTrapOptions: {
         allowOutsideClick: true,
-        // Allow outside click and let the popover manager take care of closing the popover.
+        // Allow outside click, as clicking outside closes the date picker which deactivates focusTrap.
```

Comment:
I'm wondering if we need this comment at all. [`popover`'s options](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/popover/popover.tsx#L83) don't have comments.

Reply:
I wasn't sure. Let's drop the comment then and I'll mark this as chore since it's just dropping an internal comment.